### DISCUSSION TITLE
请问数据库怎么制定

### DISCUSSION THREAD

**Discussion by hriyel** [nonebot/discussions#515] [问答]
ValueError: 必须指定一个默认数据库 (SQLALCHEMY_DATABASE_URL 或 SQLALCHEMY_BINDS[""]). 可以通过 `pip install nonebot-plugin-orm[default]` 获得开箱即用的数据库配置.

**Comment by yanyongyu** [comment]
你可以使用默认的sqlite数据库，也可以使用你想要使用的其他数据库。参考[文档](https://nonebot.dev/docs/best-practice/database/user)。

**Comment by C1en1202** [comment]
pip install nonebot-plugin-orm[default]
pip install nonebot-plugin-orm[sqlite]
这些你先得进venv再使用

**Comment by xinvxueyuan** [comment]
sqlalchemy有自己的连接池，只需要管怎么定义模型然后写数据库会话就行了

<details>
<summary>Details</summary>

tip:讨论有结果请标记为 `因已过时关闭` 或 `因已解决关闭` ，谢谢！

</details>
